presto-main/src/main/java/com/facebook/presto/sql/analyzer/FeaturesConfig.java

```diff
@@ -109,6 +109,7 @@
     private boolean iterativeOptimizerEnabled = true;
     private boolean enableStatsCalculator = true;
     private boolean ignoreStatsCalculatorFailures = true;
+    private boolean printStatsForNonJoinQuery = true;
```

Comment:
I would keep it `false` by default. Deployments with QueryEventListener that record all query plans are rather unusual, and in addition to that it is very unlikely that someone would like to see the stats for the queries that cannot be optimized by the CBO in the event log.